pkg/sync/http/http_sync.go

```diff
@@ -37,6 +37,16 @@ type Cron interface {
 	Stop()
 }
 
+func (hs *Sync) Init(ctx context.Context) error {
+	// noop
+	return nil
+}
+
+func (hs *Sync) IsReady() bool {
+	// we cannot reliably check external HTTP(s) sources
+	return true
+}
```

Comment:
Could we call the http endpoint as we are about to in the the sync anyway?

Reply:
I am not sure. I have some concerns w.r.t. the performance. Given this will eventually be used for health/ready probes, we'd hit the HTTP endpoint too frequently. Usually, probes are tested every 5s. We would do an order of magnitude more HTTP calls than what is currently configured in the cron expression

Reply:
What if we just checked it once, then cached the result? That would be similar to what is happening in the k8s sync, I think.

Reply:
I guess the other thing to consider here is that the remote server _might_ be down only temporarily during the initial fetch, which could cause a cascading startup issue.

Reply:
I adapted the impl to match the K8s one :) please, have another look @toddbaert :)

Reply:
Makes sense to me!